src/graphql/pyutils/path.py

```diff
@@ -2,15 +2,15 @@
 
 from __future__ import annotations
 
-from typing import Any, NamedTuple
+from typing import Any, NamedTuple, Optional
 
 __all__ = ["Path"]
 
 
 class Path(NamedTuple):
     """A generic path of string or integer indices"""
 
-    prev: Any  # Optional['Path'] (python/mypy/issues/731)
+    prev: Optional[Path]
```

Comment:
```suggestion
    prev: Path |  None
```

recently switched to this newer style

Reply:
Line 28 must then also be changed to `curr: Path | None = self`.

Also, in `test_executor` we must now add the line `assert prev is not None` on two lines (311, 315).

Reply:
Fixed